### PR TITLE
More compression, more awesome

### PR DESCRIPTION
**Pull request by levibuzolic**

### CODE REVIEW

File: src/components/TodoApp.jsx [hunk 1]
```diff
@@ -3,7 +3,7 @@ import '../styles/style';
 
 import React from 'react';
 import request from 'axios';
-import TodoItem from './todoItem';
```
Comment (levibuzolic) [inline src/components/TodoApp.jsx]: Bad patrik. :angry:

File: webpack.config.js [hunk 2]
```diff
@@ -91,10 +92,15 @@ module.exports = {
...
   plugins: isProduction ? [
+    new webpack.DefinePlugin({
+      'process.env': { NODE_ENV: JSON.stringify(process.env.NODE_ENV) },
+      '__DEV__': process.env.NODE_ENV !== 'production'
+    }),
     new webpack.optimize.OccurenceOrderPlugin(),
     new webpack.optimize.DedupePlugin(),
-    new webpack.optimize.UglifyJsPlugin(),
-    new CompressionPlugin()
+    new webpack.optimize.UglifyJsPlugin({ compress: { warnings: false }}),
```
Comment (levibuzolic) [inline webpack.config.js]: Shuts up those warnings, they're useless.

File: webpack.config.js [hunk 3]
```diff
@@ -91,10 +92,15 @@ module.exports = {
   },
 
   plugins: isProduction ? [
+    new webpack.DefinePlugin({
+      'process.env': { NODE_ENV: JSON.stringify(process.env.NODE_ENV) },
+      '__DEV__': process.env.NODE_ENV !== 'production'
```
Comment (levibuzolic) [inline webpack.config.js]: This will do the dead code elimination anywhere `if (__DEV__) { }` is used

File: src/components/TodoApp.jsx (hunk 1 above)
Comment (pzi) [inline src/components/TodoApp.jsx]: oopsie, I disobeyed the no relative path rule :crying_cat_face:

File: webpack.config.js (hunk 2 above)
Comment (pzi) [inline webpack.config.js]: :+1:

**Pull request merged**